pkg/applicationserver/metadata/metadata_util_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2025 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metadata_test
+
+import (
+	"context"
+	"time"
+
+	"go.thethings.network/lorawan-stack/v3/pkg/component"
+	"go.thethings.network/lorawan-stack/v3/pkg/ttnpb"
+	"go.thethings.network/lorawan-stack/v3/pkg/util/test"
+)
+
+var (
+	registeredEndDeviceIDs = &ttnpb.EndDeviceIdentifiers{
+		ApplicationIds: &ttnpb.ApplicationIdentifiers{
+			ApplicationId: "foo",
+		},
+		DeviceId: "bar",
+	}
+
+	Timeout = (1 << 7) * test.Delay
```

Comment:
Modern tests do not have global vars but define them at the beginning of the test body, including the timeout

Reply:
changed, moved them to the beginning of the test body